app/controllers/articles_controller.rb

```diff
@@ -1,10 +1,11 @@
 class ArticlesController < ApplicationController
+
```

Comment:
should have some authorization stuff here;
https://github.com/ryanb/cancan/wiki/Authorizing-controller-actions